jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/ThriftSender.java

```diff
@@ -116,7 +115,7 @@ public int flush() throws SenderException {
       throw new SenderException("Failed to flush spans.", e, n);
     } finally {
       spanBuffer.clear();
-      byteBufferSize = processBytesSize;
+      spanBytesSize = processBytesSize;
```

Comment:
shouldn't this be `spanBytesSize = 0`?

Reply:
yeah... it seems so. I'll also add some test for this